shotover-proxy/src/message/mod.rs

```diff
@@ -313,23 +318,38 @@ pub enum Value {
     None,
     #[serde(with = "my_bytes")]
     Bytes(Bytes),
+    Ascii(String),
     Strings(String),
     Integer(i64),
-    Float(f64),
+    Double(f64),
+    Float(f32),
     Boolean(bool),
     Inet(IpAddr),
     List(Vec<Value>),
     Rows(Vec<Vec<Value>>),
     NamedRows(Vec<HashMap<String, Value>>),
     Document(HashMap<String, Value>),
     FragmentedResponse(Vec<Value>),
+    Set(Vec<Value>),
+    Map(Vec<(Value, Value)>),
+    Varint(num::BigInt),
+    Bigint(num::BigInt),
```

Comment:
a cassandra bigint appears to just be an i64
https://cassandra.apache.org/doc/latest/cassandra/cql/types.html
lol

Maybe instead of bigint, tinyint etc we should name the integer variants like:
```
I64(i64)
I32(i32)
I16(i16)
I8(i8)
```
Unless the names cassandra uses happen to be standard across many DBs...?

Reply:
ahh maybe what we want is something like:
```rust
enum Value {
Int(i64, IntSize)
}


enum IntSize {
    I64,
    I32,
    I16,
    I8,
}
```

That way we dont have to special case every int type?

Reply:
Yeah I think that's a good idea.

Reply:
Okay added those changes but put all the existing Integers as I32 since that's all we had before